### PR TITLE
Add security policy as recommended by GitHub

### PR DESCRIPTION
**Pull request by beutlich**
See https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository.

### CODE REVIEW

**Comment by dietmarw** [comment]
That file makes promises that can currently with MAP-LIB being without leader ship not be fulfilled. So not much point to add this.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+| Version       | Supported          |
+| ------------- | ------------------ |
+| 4.0.x         | :white_check_mark: |
+| 3.2.3+build.x | :white_check_mark: |
+| ≤ 3.2.2       | :x:                |
+
+## Reporting a Vulnerability
+
+If you discover a potential security issue in the Modelica Standard Library you can create a public GitHub issue.
+Regular maintenance versions (containing bug-fixes) are usually released twice a year, in spring and autumn.
```
Comment (dietmarw) [inline SECURITY.md:16]:
```suggestion

```

**Review by dietmarw — APPROVED** [review]

**Review by henrikt-ma — COMMENTED** [review]
Doesn't look unreasonable to me, but I don't have enough understanding of MAP-Lib maintenance policies to say more than this.  I wish I could be of more help.

**Comment by beutlich** [comment]
> 
> 
> Doesn't look unreasonable to me, but I don't have enough understanding of MAP-Lib maintenance policies to say more than this. I wish I could be of more help.

Fair enough.

**Review by MartinOtter — APPROVED** [review]

**Pull request merged**